src/cli/main.cpp

```diff
@@ -137,7 +137,9 @@ int
 handle_download(const std::vector<std::string>& urls,
                 const std::vector<std::string>& mirrors,
                 bool resume,
-                const std::string& outfile)
+                const std::string& outfile,
+                std::string sha_cli,
```

Comment:
const std::string& is a better argument to functions :)

Reply:
Good point...

Reply:
the `const` is also important, otherwise you can't pass it on-the-fly.